starknet_py/net/client_models.py

```diff
@@ -399,6 +399,15 @@ class SierraContractClass:
     abi: Optional[str] = None
 
 
+@dataclass
+class SierraCompiledContract(SierraContractClass):
```

Comment:
It's not used currently anywhere, let's remember to plug it in / remove it.

Reply:
It is used in `create_sierra_compiled_contract`.